content/r/concepts/comments/comments.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: 'Comments'
+Description: 'A comment is a piece of text within a program that is not executed. It can be used to provide additional information to aid in understanding the code.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Comments'
+  - 'Documentation'
+CatalogContent:
+  - 'learn-r'
+  - 'paths/computer-science'
+---
+
+A comment is a piece of text within a program that is not executed. It can be used to provide additional information to aid in understanding the code.
+Many languages support single line and multi-line comments. `R` only supports single-line comments.
```

Comment:
```suggestion
A comment is a piece of text within an R program that is not executed. It can be used to provide additional information to aid in understanding the code. Many languages support single line and multi-line comments. Only single-line comments are supported in R.
```